listenbrainz_spark/utils.py

```diff
@@ -49,13 +49,12 @@ def read_files_from_HDFS(path):
         raise Py4JJavaError('An error occurred while fetching "{}": {}\n'.format(path, type(err).__name__),
             err.java_exception)
 
-def get_listens(d1, d2):
+def get_listens(current_date, adjusted_date):
```

Comment:
In this context current_date and adjusted_date make very little sense -- that context comes from the calling functions. begin_date, end_date? The docs should say if the end date is included in the results or not.

Reply:
okay. begin_date and end_date seem fine to me. Should I change the names in create_training_data_from_window also?

Reply:
maybe get_listens_for_date_range() ?

Reply:
Yes. Cool.